src/app/commands/cmd_toggle_timeline_thumbnails.cpp

```diff
@@ -0,0 +1,50 @@
+// Aseprite
+// Copyright (C) 20016  Carlo "zED" Caputo
```

Comment:
You can remove the "zED" part from copyright (I'm not sure about the legal validity about that)